tree/30_generic_methods/command_execution_once.cf

```diff
@@ -118,11 +118,12 @@ bundle agent command_execution_once(command, ok_codes, until, unique_id)
 
     # Problem when writing the flag
     pass3.reserved.!set_ok::
-      "failure" usebundle => _classes_failure("${old_class_prefix}");
-      "failure" usebundle => _classes_failure("${class_prefix}");
+      "failure"                  usebundle => _classes_failure("${class_prefix}");
+      "${report_data.method_id}" usebundle => _classes_failure("${report_data.method_id}");
```

Comment:
what I did for the methods I migrated was to systematically use the method_id for all usebundle promisers

Reply:
that's not enough - the method don't report